README.md

```diff
@@ -289,21 +259,30 @@ Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
     { "name", "Foo" }
 };
 ```
-
 Otherwise the parameter can be omitted or `null` can be passed in its place.
 
-#### `reference`
+##### `reference`
 
```

Comment:
should match sms wording
An optional identifier you generate. The reference can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.

You can omit this argument if you do not require a reference for the notification.